### DISCUSSION TITLE
Giving good feedback

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#578] [General Resources]
We love good feedback, and we're focusing on that for our June monthly challenge. 

Whether it's feedback in a blog, a Pull Request, code, or other content, having good feedback helps us to grow. 

What are some good resources for giving good feedback? (It doesn't have to be tech specific--we love _Radical Candor_!)

**Comment by BekahHW** [comment]
https://www.iamdeveloper.com/posts/how-i-review-pull-requests-44nl/

**Reply by mccurcio** [reply]
Great link and article! TY
